fastlane_bot/bot.py

```diff
@@ -636,6 +636,7 @@ def validate_pool_data(self, arb_opp):
                 "tkn1_symbol": current_pool.tkn1_symbol,
                 "tkn0_decimals" : current_pool.tkn0_decimals,
                 "tkn1_decimals": current_pool.tkn1_decimals,
+                "descr": current_pool.descr
```

Comment:
1. Can it yield an undesired result somewhere, if the value of `current_pool.descr` is, for example, `None`?
2. Can you please check if `"last_updated_block"` also happens to be needed here?